modules/cnv-openshift-client-commands.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-using-the-cli-tools.adoc
+
+[id="cnv-openshift-client-commands_{context}"]
+= {product-title} client commands
+
+The {product-title} `oc` client is a command-line utility for managing 
+{product-title} resources. The following table contains the *oc* commands
+ used throughout the {ProductName} documentation.
+
+.`oc` commands
+
+[width="100%",cols="42%,58%",options="header",]
+|===
+|Command |Description
+
+|`oc login -u <user_name>`
+|Log in to the {product-title} cluster as `<user_name>`.
+
+|`oc get <object_type>` 
+|Display a list of objects for the specified
+object type in the project.
+
+|`oc describe <object_type> <resource_name>` 
+|Display details of the
+specific resource.
```

Comment:
Should we add _in the project_ here? (I constantly look for objects in the wrong project and get frustrated to see the `<resource> not found` messages!)